### ISSUE TITLE
[FEATURE REQUEST] Including more analytics regarding the repositories

### ISSUE THREAD

**Issue by qwadrox** [fluttergems/fluttergems#290]
Have you considered enhancing the analytics capabilities for repositories?

Flutter Gems effectively tackles longstanding community challenges, acting as a crucial solution across various aspects. I'm not alone in recognising the value of improved analytics for packages, especially as the package ecosystem continues to expand. This could be another aspect. 

Flutter Gems already has some status about the repos: 
<img width="201" alt="image" src="https://github.com/fluttergems/fluttergems/assets/33902750/ffe66da0-2b8a-45b7-8d02-564caa1c41f6">

<img width="496" alt="image" src="https://github.com/fluttergems/fluttergems/assets/33902750/c987406d-664b-42e1-95f8-66862b1dfe73">

https://www.linkedin.com/posts/dane-mackier-a0b99670_the-worst-part-of-the-flutter-ecosystem-is-activity-7171866992050692096-BMmy?utm_source=share&utm_medium=member_desktop

**But integrating features, kinda like what analyzemyrepo does, could greatly enhance the platform's utility:**
https://github.com/CrowdDotDev/analyzemyrepo

In summary, I would be happy to see more metrics about the packages. 

Thank you for the platform!
